### DISCUSSION TITLE
Dialog - change backgroup color

### DISCUSSION THREAD

**Discussion by tatwww** [buefy/buefy#4085] [Q&A] [answered]
Does Buefy provide a function to change the background color? It is difficult to see the background content on a black background. What can I do?

![image](https://github.com/user-attachments/assets/21f696fc-929a-47c0-b84b-9f9eda4d11b1)

**Comment by andrei-dracea** [comment, marked as answer]
set it via bulma variable `$modal-background-background-color` (see [documentation](https://versions.bulma.io/0.9.4/documentation/components/modal/#variables))

**Comment by wesdevpro** [comment]
> set it via bulma variable `$modal-background-background-color` (see [documentation](https://versions.bulma.io/0.9.4/documentation/components/modal/#variables))

This is the correct answer. There are multiple methods of doing this if for some reason you need a different color elsewhere you can always used scope css or sass to manually change the background
